assets/javascripts/views/search/search.coffee

```diff
@@ -138,6 +141,10 @@ class app.views.Search extends app.View
     $.stopEvent(event)
     return
 
+  onScopeChange: =>
+    @value = ''
```

Comment:
Took me a while to figure out why we need this. Instead of doing `$.trigger @input, 'input'` below I would move all the logic for "re-trigger search after scope change" in here so it's a bit more clear:

```coffee
onScopeChange: =>
  @value = '' # reset memoized value to re-run the current search (with the new scope)
  @onInput()
```